test/js/bun/shell/exec.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as t from "bun:test";
+import { TestBuilder } from "./test_builder";
+import { bunEnv } from "harness";
+
+const BUN = process.argv0;
+
+t.describe("bun exec", () => {
```

Comment:
we don't use `t` anywhere else in the codebase, lets avoid adding this pattern

```suggestion
describe("bun exec", () => {
```